lib/types/user-types.js

```diff
@@ -2,9 +2,18 @@
 
 import PropTypes from 'prop-types';
 
+type RelationshipStatus =
+  | 'request_sent'
+  | 'request_received'
+  | 'blocker'
+  | 'blocked'
+  | 'both_blocked'
+  | 'friends'
+  | 'know_of';
 export type UserInfo = {|
   id: string,
   username: ?string,
+  relationshipStatus?: RelationshipStatus,
```

Comment:
I think we need to update the following as well:

- `userInfoPropType`
- `AccountUserInfo`
- `accountUserInfoPropType`
- `RelativeUserInfo`
- `relativeUserInfoPropType`